src/sdk/namespace_blob.js

```diff
@@ -656,6 +668,17 @@ class NamespaceBlob {
         const xattr = _.extend(obj.metadata, {
             'noobaa-namespace-blob-container': this.container,
         });
+
+        // Restore invalid metadata keys
+        for (let tag_key in xattr) {
+            if (tag_key.startsWith("rename_key_")) {
+                let valid_key = tag_key.substring(11);
```

Comment:
here I think it's better to use slice, and rename_key_.length() instead of hard coded 11

Reply:
you can just declare the size of it as a const at the beginning of the file/ outside of the loop
also instead of slice, have you considered using replace one with '' (empty string ? )